include/ncpp/Plane.hh

```diff
@@ -794,9 +789,9 @@ namespace ncpp
 			return ncplane_set_scrolling (plane, scrollp);
 		}
 
-		void styles_set (CellStyle styles) const noexcept
+		void set_attr (CellStyle styles) const noexcept
```

Comment:
set_attr() ought take unsigned; styles_set() ought be restored and take CellStyle.